src/aliases/store/postgres/alias.go

```diff
@@ -23,10 +23,13 @@ func NewAlias(pgClient postgres.Client) *AliasStore {
 }
 
 func (s *AliasStore) CreateAlias(ctx context.Context, registry aliasent.RegistryName, alias aliasent.Alias) (*aliasent.Alias, error) {
-	a := aliasmodels.AliasFromEntity(alias)
+	a, err := aliasmodels.AliasFromEntity(alias)
```

Comment:
This function should not return an error at this level.  Same comment for all functions of this Agent/Repository

Reply:
So what should we do about the error that could be raised inside it?

Reply:
I removed the error from the func as we don't need to marshal to JSON anymore.